src/core/server/saved_objects/routes/find.ts

```diff
@@ -59,6 +59,9 @@ export const registerFindRoute = (router: IRouter) => {
           namespaces: schema.maybe(
             schema.oneOf([schema.string(), schema.arrayOf(schema.string())])
           ),
+          workspaces: schema.maybe(
```

Comment:
Could we reuse `formatWorkspaces` and `workspacesValidator` in utils?

Reply:
Sure.